README.md

```diff
@@ -204,3 +204,17 @@ bx service delete "Compose for PostgreSQL-GL"
 
 # License
 [Apache 2.0](LICENSE)
+
+# Privacy Notice
+
+Sample Kubernetes Yaml file that includes this package may be configured to track deployments to [IBM Bluemix](https://www.bluemix.net/) and other Kubernetes platforms. The following information is sent to a [Deployment Tracker](https://github.com/IBM/metrics-collector-service) service on each deployment:
+
+* Kubernetes Cluster Provider(`Bluemix,Minikube,etc`)
+* Kubernetes Machine ID (`MachineID`)
```

Comment:
I updated the kubernetes client yesterday to collect more information, so we need to use the newest version of the privacy notice at https://github.com/IBM/metrics-collector-client-kubernetes#privacy-notice